Lib/build.sh

```diff
@@ -1 +1,7 @@
-pyinstaller --noconsole --name "TruFont" --onefile -i "defconQt/resources/app.ico"  defconQt/__main__.py
+pyinstaller \
+	--noconsole \
+	--name "TruFont" \
+	--onefile \
+	--icon="defconQt/resources/app.ico" \
+	--osx-bundle-identifier=io.github.trufont \
+	TruFont.spec;
```

Comment:
Does this work with the other platforms? Maybe this file should be changed into a python script.